src/tests/eventpipe/EventPipe.UnitTests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
I looked at the failing tests and it looks like one loader event we test for (AssemblyUnload) is getting fired under a different keyword now. If you want to skip the EventPipe tests for now, I think that's fine for now and we can file an issue to investigate that issue further.